src/content/impact/ImprovedBiodiversity.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Increased Carbon Sequestration and Storage
+image: ./dummy.svg
+---
+
+Protect or restore the variety of life at genetic, species and ecosystem scales, in an area, including different species of plants, animals, and other organisms.
```

Comment:
the definition is correct for the md Improved Biodiversity but the title says increased carbon sequestration and storage

Reply:
suyggested change above

Reply:
I changed this to the correct title